### ISSUE TITLE
Required ARIA parent role is not present for the tab items in the page

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#590]
Note: Same issue reproes throughout the application.

Impact on a Person with Disability (PWD):
Visually challenged users who rely on screen reader for navigation are facing difficulty as Required ARIA parent role is not present for the tab items in the page.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Tools: Accessibility insights for web

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to the collect tab and invoke it. Run AI.
Step 4: Verify whether required ARIA parent role is present for the tab items in the page or not

Actual Result: 
Required ARIA parent role is not present for the tab items in the page.
Current Observation: 
1. When focus lands on the collect tab NVDA is narrating as "Tab selected workflow collect"

Expected Result:
Required ARIA parent role should be present for the tab items in the page.
Ex: 
1. When focus lands on the collect tab NVDA should be narrating as "Tab selected workflow collect list 1 of 4"

Suggestive fix
1.provide 'aria-setsize="4", 'aria-posinset="1"' for
first tab.
2.provide 'aria-setsize="4", 'aria-posinset="2"' for
second tab.
3.provide 'aria-setsize="4", 'aria-posinset="3"' for
third tab.
4.provide 'aria-setsize="4", 'aria-posinset="4"' for
fourth tab.

K4W:
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require parent roles are contained by them (.active)

Tags: Accessibility, WCAG 1.3.1, aria-required-parent

Issue: Ensures elements with an ARIA role that require parent roles are contained by them (aria-required-parent - https://accessibilityinsights.io/info-examples/web/aria-required-parent)

Target application: Retrospectives - Boards - https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_apps/hub/ms-devlabs.team-retrospectives.home

Element path: .external-content--iframe;.active

Snippet: <div class="retrospective-workflowState active" aria-label="Selected Collect workflow stage" role="tab" tabindex="0"><p class="stage-text">Collect</p></div>

How to fix:
Required ARIA parent role not present: tablist.

MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)
 
WCAG Reference:
[https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.w3.org%2fWAI%2fWCAG21%2fUnderstanding%2finfo-and-relationships.html&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671698366431%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=%2fEKPR2a5eIbXGSOloeJQVNFeOxwoYuSI2zFZJZfoRWs%3D&reserved=0)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713019
